cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -289,6 +289,8 @@ class fixed_point {
   scale_type _scale;
 
  public:
+  using representation_type = Rep;
```

Comment:
Looking at other wrapper types like `std::chrono::duration`, I think `rep` is more canonical for this member typedef. 
```suggestion
  using rep = Rep;
```

Reply:
Done.